images/tier1/navigate-dark.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
That's likely the case, it had conflicted so i renamed the new one.  I hadn't realized we were updating some of these